compliance_checker/cf/cf.py

```diff
@@ -130,14 +130,19 @@ def setup(self, ds):
 
         :param netCDF4.Dataset ds: An open netCDF dataset
         """
-        self._find_coord_vars(ds)
+        self.coord_vars = self._find_coord_vars(ds)
```

Comment:
This is unused due to the distinction between "coordinate variables" (variables with a single dimension that has the same name as the variable) and "coordinate data variables", which in my interpretation appear to be variables referenced by the `coordinates` attribute in other variables.